### DISCUSSION TITLE
Connecting to MS SQL server express (LocalDB)

### DISCUSSION THREAD

**Discussion by GGAlanSmithee** [prisma/prisma#11536] [Q&A] [answered]
Hi,

I have gotten a connection to work with a local SQL Server using port 1433 with integrated security, but I cannot get this, or any other sort of connection, to work with SQL Server Express (LocalDB).

Is this supported?

Thanks!

Links for context:

* https://github.com/prisma/prisma/discussions/11492
* [Stack Overflow](https://stackoverflow.com/questions/70928150/authentication-failed-for-user-when-running-prisma-db-pull-on-a-sql-server-dat)

**Comment by pantharshit00** [comment, marked as answer]
Hello @GGAlanSmithee , our apologies for the delay in getting back to you! Due to the large number of questions in our community channels, unfortunately this slipped through the cracks. It's very important for us to address your question, so let us know in a comment if this is still an open discussion. 


Looks like this was answered in the [other discussion you have opened](https://github.com/prisma/prisma/discussions/11492). Let us know if you facing any more trouble.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
